docs/docs/api-reference/starlark-reference/plan.md

```diff
@@ -570,6 +570,15 @@ The `run_python` instruction executes a one-time execution task. It runs the Pyt
             "/coinbase/address.json"
         ],
 
+        # If the recipe returns a code that does not belong on this list, this instruction will fail.
+        # OPTIONAL (Defaults to [0])
+        acceptable_codes = [0, 0], # Here both 0 and 1 are valid codes that we want to accept and not fail the instruction
+
+        # If False, instruction will never fail based on code (acceptable_codes will be ignored).
+        # You can chain this call with assert to check codes after request is done.
+        # OPTIONAL (Defaults to False)
+        skip_code_check = False,
```

Comment:
IMO a simpler way to implement this would have been to have `acceptable_codes` take in `None` to mean "accept every code" rather than adding a new parameter, but it's in now

Reply:
agree w u, i was following the pattern that `plan.exec` has and think its better if all instructions follow the same pattern